spec/integration/puppet_x/puppetlabs/scheduled_task/taskscheduler2_v1task_spec.rb

```diff
@@ -39,6 +39,9 @@
       expect(subjectv2.parameters).to eq(subjectv1.parameters)
       expect(subjectv2.application_name).to eq(subjectv1.application_name)
       expect(subjectv2.trigger_count).to eq(subjectv1.trigger_count)
+      # NOTE: this currently only works on a system with UTC dates because
```

Comment:
This may mean that V1 hash should use local time instead of UTC. Will have to revisit prior to merge.